lib/new_relic/agent/instrumentation/grpc/server/instrumentation.rb

```diff
@@ -12,29 +12,32 @@ module GRPC
         module Server
           include NewRelic::Agent::Instrumentation::GRPC::Helper
 
+          DT_KEYS = [NewRelic::NEWRELIC_KEY, NewRelic::TRACEPARENT_KEY, NewRelic::TRACESTATE_KEY].freeze
```

Comment:
These need to be removed from the attributes hash that is being populated for the server. The NR stuff is already successfully being used to set DT headers.